boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_defconfig

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+
+CONFIG_MAIN_STACK_SIZE=2048
+CONFIG_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_UART_CONSOLE=y
+CONFIG_I2C=y
+CONFIG_GPIO=y
+#CONFIG_GPIO_AW9523B=y
+CONFIG_REGULATOR=y
```

Comment:
The regulator function has not been implemented.
It is necessary to remove unimplemented features.

Please consider applying https://github.com/zephyrproject-rtos/zephyr/pull/79765/commits/a59a4dbc9f595ca73b93f04b3f37918bd343f267.

Reply:
There is an another pr #78098 for that regulator.

Reply:
It it not yet available.
If you try to introduce feature releases in the future in advance, it will cause inconsistencies.
So please add this line after the driver is introduced.
We should remove this line at this time.

Reply:
> It it not yet available. If you try to introduce feature releases in the future in advance, it will cause inconsistencies. So please add this line after the driver is introduced. We should remove this line at this time.

Okay, Removed